app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    """Class add new car which want to make wash!"""
+
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power,
+                 average_rating, count_of_ratings):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars):
+        cars_nums = 0
+        for car in cars:
+            if car.clean_mark <= self.clean_power:
+                cars_nums += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
+            else:
+                cars_nums += 0
+        return round(cars_nums, 1)
+
+    def calculate_washing_price(self, cars):
+        a = cars.comfort_class * (self.clean_power - cars.clean_mark)
+        income = a * self.average_rating / self.distance_from_city_center
+
+        return round(income, 1)
+
+    def wash_single_car(self, cars):
```

Comment:
```suggestion
    def wash_single_car(self, car):
```
The method takes only one car

Reply:
corrected